Added Mar 2021 Newsletter (#148)
pythonpune/meetup-talks: content/2021/March/community_news.md

* 2021 State Of Pyladies 
  https://reshamas.github.io/2021-state-of-pyladies/

* Python 3.10 Pattern Matching in Action 
  https://www.youtube.com/watch?v=SYTVSeTgL3s

* Using Python Notebook to analyze news 
  https://www.youtube.com/watch?v=paVSRMTitLQ

* Friendly-traceback: testing with Real Python 
  https://aroberge.blogspot.com/2021/02/friendly-traceback-testing-with-real.html

* Ultimate Python Quickstart Guide 
  http://dabeaz.blogspot.com/2009/10/ultimate-python-quickstart-guide.html

* Typer, build great CLIs. Easy to code. Based on Python type hints. 
  https://typer.tiangolo.com/

## New projects

* shelmet, A shell power-up for working with the file system and running subprocess commands 
  https://shelmet.readthedocs.io/

* dnc, A CLI tool to check domain names configuration 
  https://github.com/fcambus/dnc

* jut, Render Jupyter notebook in the terminal 
  https://github.com/kracekumar/jut/

* certifi-system-store, a certifi hack to use system trust store on Linux and FreeBSD 
  https://github.com/tiran/certifi-system-store

## Help Needed

* PythonPune meetup user group is looking for speakers for monthly meetup talks,

>   Please submit talks here: https://github.com/pythonpune/meetup-talks/issues

Reviewer: ```suggestion
  Please submit talks here: https://github.com/pythonpune/meetup-talks/issues/new/choose
```

* If you contribute to open-source projects, organize a meetup, help with the community,
  or basically spend 5 or more hours a month making the Python community better and
  aren't already a PSF member, you can self-certify and join for free! We would like that. 
  https://www.python.org/psf/membership/

## Upcoming events in & around Pune

* CFP for PyCon India 2021 is open now 
  https://in.pycon.org/blog/2021/cfp-announcement.html

## Thanks

> * Mar PythonPune meetup 2021 is collaborated with BangPypers

Reviewer: ```suggestion
* March PythonPune meetup 2021 is in collaboration with BangPypers
```

>   Thanks to Abhiram, Anirudha, Ritesh, Bhavin, Akshay, Vishal for making it. 

Reviewer: ```suggestion
  Thanks to Abhiram, Anirudha, Ritesh, Bhavin, Akshay, Vishal for making it possible. 
```
